pyxem/signals/common_diffraction.py

```diff
@@ -162,3 +162,23 @@ def get_integrated_intensity(self, roi, out_signal_axes=None):
         dark_field_sum.metadata.set_item("Diffraction.intergrated_range", roi_info)
 
         return dark_field_sum
+
+    def add_navigation_signal(self, data, name="nav1", unit=None, nav_plot=False):
```

Comment:
Sorry, being slow, can you explain to me the use case for this?

Reply:
As with above it probably deserves it's own pull request. I was just trying to map out how everything would work.  Let me comment on it below.